commit bb5ef1019556f5a6f05ae53274b5f481e9a71b0b (refs/changes/52/2252/2)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2017-08-11T20:47:58+00:00 (2 years, 2 months ago)
    
    HEAD-2364 dhcpd zone runs aground in default RSS cap
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/etc/defaults.json b/etc/defaults.json
index b4a508e..fa10bb6 100644
--- a/etc/defaults.json
+++ b/etc/defaults.json
@@ -70,6 +70,13 @@
         "ufds": "20140728T173339Z"
     },
     "updatedSizeParameters": {
+        "dhcpd": {
+            "max_physical_memory": 512,
+            "max_locked_memory": 512,
+            "max_swap": 1024,
+            "cpu_cap": 200,
+            "package_name": "sdc_512"
+        },
         "manatee": {
             "max_lwps": 4000
         },
